R/data.R

```diff
@@ -484,6 +484,46 @@
 #'
 "delta_total_diverted"
 
+#' @title Flow at Freeport
+#' @description flow in cms at Freeport (C400)
+#' @format A matrix with months (1-12) as rows and years (1980-1999) as columns
+#' @details
+#' The flows here are obtained from a CALSIM run designed for the structured
```

Comment:
It would be helpful to specify which nodes are being used to represent the different flows and exports

Reply:
I have it shown as part of the description, I agree I should make more explicit....should it stay in the description or be moved to the details section?

Reply:
Ahhh, i see it now... I think in the description is fine, just some text like "Freeport flows are represented by CALSIMII 'FLOW-CHANNEL' node C400. Flows have been converted from cfs to cms for model usage."